integration_tests/testdata/unsigned_msgs/bank_send_unsigned.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
this file seems unused. Is it referenced somewhere?

Reply:
it's more for structure and doc reference as to where the Raw functions came from, we can remove the unsigned_msgs directories altogether